dist/api_err.py

```diff
@@ -188,6 +188,10 @@ def __init__(self, name, value, desc, long_desc=None, **flags):
         "Another thread currently holds the checkpoint lock", '''
         This sub-level error indicates that a concurrent operation is performing
         a checkpoint.'''),
+    Error('WT_CONFLICT_LIVE_RESTORE', -32013,
+        "Conflict performing operation due to running live restore", '''
+        This sub-level error indicates that there is conflict perform the operation
```

Comment:
`...there is conflict perform...` -> `...there is a conflict performing...`
Add `a` and `performing`